### DISCUSSION TITLE
Unable to source env variables

### DISCUSSION THREAD

**Discussion by i4lina** [prisma/prisma#11714] [Q&A] [answered]
Hi, I'm trying to deploy a nodejs app on a dokku server. The server automatically created a DATABASE_URL env variable that stores the url needed to connect with the database on the remote machine. In my local machine, for developing purpose, i use a .env file containing the local url to the database. I removed this file so that Prisma could source the right env variable from the sistem, however when I try running the migration I get this error.
```Error: Get config: Schema Parsing P1012

error: Environment variable not found: DATABASE_URL.
  -->  schema.prisma:7
   | 
 6 |   provider = "postgresql"
 7 |   url      = env("DATABASE_URL")
   | 

Validation Error Count: 1

error Command failed with exit code 1.
```
Is it possible that prisma cannot access the system's env variables? can i pass the link via prisma cli?

**Comment by janpio** [comment, marked as answer]
No, if the CLI tells you the environment variable was not found, then it is not set for the context that is running Prisma CLI.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
